torchmultimodal/modules/layers/codebook.py

```diff
@@ -213,3 +207,30 @@ def extra_repr(self) -> str:
         return "num_embeddings={}, embedding_dim={}".format(
             self.num_embeddings, self.embedding_dim
         )
+
+    def _load_from_state_dict(
```

Comment:
Would probably adding a unit test here with fake inputs.
Reason being that it will give us early signal about not just the overloading logic but also the validity of the pytorch internal API.

Reply:
how about a unit test where I have a parent module that has `Codebook` as a child module, and I load in a fake state dict to see if this is still called and the flag is set?

Reply:
Sounds good. Parent module -- you mean an instance of VQVAE?